README.md

```diff
@@ -22,6 +22,12 @@ The below describes building Frankendancer from scratch and running it
 optimized on a stock Linux image. You will need basic development tools
 like `make`, `gcc` along with `rustc`, and `clang`.
 
+Officially supported compilers for production use are the latest releases of _gcc 11_, _gcc 12_ and _gcc 13_.
+Other compilers are likely to work but are not tested or secured by the Firedancer team.
+
+
+The latest release of Clang 17 is supported for development and tooling.
```

Comment:
I think better to just say "Building with clang is supported for static analysis and other development use cases, but not recommended for production."

Reply:
I don't think that we're doing static analysis with clang (outside of the checks that are implicit to building).